resources/rgbscripts/flames.js

```diff
@@ -0,0 +1,172 @@
+/*
+  Q Light Controller Plus
+  empty.js
+
+  Copyright (c) Branson Matheson
+  Based on work https://slicker.me/javascript/fire/fire.htm
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0.txt
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+// Development tool access
+var testAlgo;
+
+(
+  function()
+  {
+    var util = new Object;
+    util.initialized = false;
+
+    var algo = new Object;
+    algo.apiVersion = 2;
+    algo.name = "Flames";
+    algo.author = "Branson Matheson";
+    algo.acceptColors = 1;
+
+    algo.properties = new Array();
+    algo.fireAmount = 40;
+    algo.properties.push("name:fireAmount|type:range|display:Amount|values:1,200|write:setAmount|read:getAmount");
+    algo.getAmount = function() {
+      return algo.fireAmount;
+    };  
+    algo.setAmount = function(_size) {
+	    // Only set if the input is valid.
+      if (!(parseInt(_size) === NaN) && parseInt(_size) > 0) {
+        algo.fireAmount = parseInt(_size);
+        util.initialized = false;
+      }
+    };
+
+    algo.fireDepth = 40;
+    algo.properties.push("name:fireDepth|type:range|display:Depth|values:1,100|write:setDepth|read:getDepth");
```

Comment:
Codacy has a fix for the issue: [Replace `····algo.properties.push("name:fireDepth|type:range|display:Depth|values:1,100|write:setDepth|read:getDepth"` with `··algo.properties.push(⏎····"name:fireDepth|type:range|display:Depth|values:1,100|write:setDepth|read:getDepth"⏎··`](https://app.codacy.com/gh/mcallegari/qlcplus/pullRequest?prid=9305478)
```suggestion
  algo.properties.push(
```